airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricTags.java

```diff
@@ -26,7 +26,11 @@ public static String getReleaseStage(final ReleaseStage stage) {
   }
 
   public static String getFailureOrigin(final FailureOrigin origin) {
-    return origin.value();
+    if (origin != null) {
+      return origin.value();
+    } else {
+      return null;
+    }
```

Comment:
I know this method is preexisting, but it feels unnecessary.  The `FailureOrigin` is provided by the caller and this simply calls a method on the provided class?

Additionally this method is [only called once](https://github.com/airbytehq/airbyte/blob/3d17ea07417201c079b3bf7edc733ca1ecf85a92/airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java#L265-L268):
```
for (final FailureReason reason : failureSummary.getFailures()) {
  MetricClientFactory.getMetricClient().count(OssMetricsRegistry.ATTEMPT_FAILED_BY_FAILURE_ORIGIN, 1,
    new MetricAttribute(MetricTags.FAILURE_ORIGIN, MetricTags.getFailureOrigin(reason.getFailureOrigin())));
}
```

Seems that we could move this functionality to this^ spot instead of having a dependency on `MetricTags`.  Unless there is something I am missing here?

Reply:
This was from when I was testing and running into odd errors - I'm not sure this actually ever happens and I can remove this since it's unrelated anyway!